mozilla/blurts-server
# MNTOR-2761

# References:

Jira: MNTOR-2761

# Description
Resolving style issues for landing page on mobile.  Note:  Was unable to recreate any of the issues described in ticket except for size inconsistencies for close icons on FAQ section ([see ticket for details](https://mozilla-hub.atlassian.net/browse/MNTOR-2761?focusedCommentId=905817)) so scope was reduced to that single issue.  For this, the style bug was that the icons were different sizes from each other and this was made worse as the screen width was reduced.

# How to test
Navigate to FAQ section of landing page.  Using dev tools or browserstack, emulate mobile screen sizes and confirm the close icons are maintaining consistent sizing between them.

# Checklist (Definition of Done)
